docs/specs/parsing.md

```diff
@@ -13,18 +18,75 @@
     - `DateTime`, `DateTimeOffset`
     - `TimeSpan`
     - `Guid`
+    - `Uri`
 * Parses different cultures
     - e.g. English and German dates
-    - Needs to handle all cultures .NET can handle
+    - Needs to handle all cultures .NET Core can handle
+* Parse any outputs produced by [formatting](formatting.md)
+    - This ensures parsing & formatting behave symmetric
 * Parse from these encodings
-    - UTF8      <------------- THATS WHAT WE CARE ABOUT
+    - UTF8
     - ASCII
     - UTF16-BE
     - UTF16-LE
 * Parses from existing buffers without knowing up-front how many bytes the
   parsing needs to consume
```

Comment:
This section should also talk about the goals for supporting all "formats", i.e. 'G', 'N', "x", "X", etc. and more broadly about compatibility with .NET, i.e. can this library parse everything that the current APIs can parse with the same results?